File: src/main/java/logan/pickpocket/commands/StatusCommand.java
package logan.pickpocket.commands;

import logan.api.command.BasicCommand;
import logan.api.command.SenderTarget;
import logan.pickpocket.config.Config;
import org.bukkit.entity.Player;

import java.util.Collections;

public class StatusCommand extends BasicCommand<Player> {

    public StatusCommand() {
        super("status", "pickpocket.status", 0, 0,
                new String[0], SenderTarget.PLAYER, "pickpocket",
                Collections.emptyList(),
                "Usage:\n/pickpocket status");
    }

    @Override
    public boolean run(Player sender, String[] args, Object data) {
        sender.sendMessage("Pickpocketing is " + Config.isPickpocketingEnabled() + ".");
        return true;
    }
}

File: src/main/java/logan/pickpocket/commands/StatusCommand.kt
package logan.pickpocket.commands

import logan.api.command.BasicCommand
import logan.api.command.SenderTarget
import logan.pickpocket.config.Config
import org.bukkit.entity.Player

class StatusCommand : BasicCommand<Player>(
    "status",
    parentCommand = "admin",
    target = SenderTarget.PLAYER,
    permissionNode = "pickpocket.admin.status",
    usage = """
        Usage:
        /pickpocket admin status (no args)"
    """.trimIndent()
) {

    override fun run(sender: Player, args: Array<out String>, data: Any?): Boolean {
        sender.sendMessage("Pickpocketing is ${Config.isPickpocketingEnabled}.")
        return true
    }
}